nolang/builtins/buffer.py

```diff
@@ -0,0 +1,11 @@
+from nolang.builtins.spec import parameters
+from nolang.objects.int import W_IntObject
+from nolang.objects.unicode import W_StrObject
+
+
+@parameters(name='buffer')
+def buffer(space, args_w):
```

Comment:
here it should raise some exception when it's none of those. We don't have mechanisms yet, so it's ok to raise AppError from space.w_exception or something like that (or a fatal exception for now)